docs/content/_navigation.json

```diff
@@ -44,6 +44,10 @@
         "title": "Building an asset graph",
         "path": "/tutorial/building-an-asset-graph"
       },
+      {
+        "title": "Scheduling your pipeline",
+        "path": "/tutorial/scheduling-your-pipeline"
```

Comment:
Do we need to update the link at the end of the prior step to point to this one?